test/com/facebook/buck/android/AndroidLibraryIntegrationTest.java

```diff
@@ -51,4 +51,20 @@ public void testAndroidLibraryDoesNotUseTransitiveResources() throws IOException
     result.assertFailure();
     assertTrue(result.getStderr().contains("package R does not exist"));
   }
+
+  @Test
```

Comment:
All the other scala integration tests have increased timeouts because of how long they take on CI. Adjust these accordingly as well.